src/lexer.l

```diff
@@ -122,6 +122,7 @@ struct lexer_param;
 
 ([a-zA-Z_][a-zA-Z_0-9]*::)*[a-zA-Z_][a-zA-Z_0-9]*  { yylval->literal = jv_string(yytext); return IDENT;}
 \.[a-zA-Z_][a-zA-Z_0-9]*  { yylval->literal = jv_string(yytext+1); return FIELD;}
+\$[a-zA-Z_][a-zA-Z_0-9]*  { yylval->literal = jv_string(yytext+1); return VARIABLE;}
```

Comment:
Should variable/labels also have `([a-zA-Z_][a-zA-Z_0-9]*::)*` like for functions?
Should I still allow `$   foo` with spaces?

Reply:
> Should variable/labels also have `([a-zA-Z_][a-zA-Z_0-9]*::)*` like for functions?

Yes, otherwise some tests fail :)

Reply:
I don't care about `$    foo` -- we've never documented that that works, so I don't mind breaking that.

Reply:
> Should variable/labels also have ([a-zA-Z_][a-zA-Z_0-9]*::)* like for functions?

Yes because one can have namespaced binding names via `import`.